bin/yb-ctl

```diff
@@ -1520,18 +1549,19 @@ class ClusterControl:
         title = "Node {}: {}".format(daemon_index, ", ".join(process_list))
         self.print_status_box(title, info_kv_list)
 
-    def for_all_daemons(self, fn, server_counts_map=None):
+    def for_all_daemons(self, fn):
         """
         Run the given function for all daemons.
+
+        :returns: return values of each function call.
+        :rtype: dict
         """
+        fn_returns = {daemon_type: [] for daemon_type in DAEMON_TYPES}
         for daemon_type in DAEMON_TYPES:
-            if server_counts_map:
-                num_servers = server_counts_map[daemon_type]
-            else:
-                num_servers = self.get_number_of_servers(daemon_type)
-
+            num_servers = self.get_number_of_servers(daemon_type)
```

Comment:
wait, how does this work during `create`?? we won't have the respective directories created per node, for `get_number_of_servers` to actually be able to return the correct numbers...

Reply:
It goes into the first if case, which uses replication factor.